### PR TITLE
(feat):Made dashboard section responsive for mobile screen.

### PR DESCRIPTION
**Pull request by rayAritra**
updated the page.jsx file for dashboard section and made it responsive for mobile screen

## Title
made the dashboard section responsive for mobile screen.

## Screenshots
 *provide an
Screenshot 2024-05-20 at 10 20 45 AM

### CODE REVIEW

**Comment by Arindam200** [comment]
Please make the check boxes aligned in one line. 

Rest looks good!

File: src/app/dashboard/page.jsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ export default function page() {
 
   const isAdmin = sessionClaims?.metadata?.role === "admin";
   return (
-    <div className="h-full flex">
-      <div className="flex flex-col justify-between basis-1/12 pt-28 bg-[#FFFFFF05] border-r-2 border-[#FFFFFF20]">
+    <div className="h-full flex flex-col lg:flex-row">
+      <div className="flex flex-col justify-between lg:basis-1/12 pt-28 bg-[#FFFFFF05] lg:border-r-2 border-[#FFFFFF20]">
```
Comment (Swpn0neel) [inline src/app/dashboard/page.jsx:19]: Add `max-lg:border-b-2`

File: src/app/dashboard/page.jsx [hunk 2]
```diff
@@ -62,9 +62,9 @@ export default function page() {
           </div>
         </div>
         <Tabs />
-        <div className="h-full flex border-2 border-[#FFFFFF20] rounded-lg">
-          <div className="flex flex-col basis-1/3 justify-between py-8 pl-10">
-            <div className="">
+        <div className="h-full flex flex-col lg:flex-row border-2 border-[#FFFFFF20] rounded-lg">
```
Comment (Swpn0neel) [inline src/app/dashboard/page.jsx:65]: Almost all `flex-col lg:flex-row` in the whole code can be replaced with `max-lg:flex-col`. Try doing that, and check the results!!

File: src/app/dashboard/page.jsx [hunk 3]
```diff
@@ -107,27 +107,25 @@ export default function page() {
                 />
               </FormGroup>
             </div>
-            <div className="">
+            <div>
               {isAdmin && (
```
Comment (Swpn0neel) [inline src/app/dashboard/page.jsx:111]: Keep the whole code inside the parenthesis within <> and </>. Don't remove it!!

**Pull request merged**